sdk/include/opentelemetry/sdk/trace/span_limits.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace trace
+{
+struct SpanLimits
+{
+	int AttributeCountLimit = 128;
+	int EventCountLimit = 128;
+	int LinkCountLimit = 128;
+	int AttributePerEventCountLimit = 128;
+	int AttributePerLinkCountLimit = 128;
+};
+
+} // namespace trace
+} // namespace sdk
+
```

Comment:
Can you this be a class with static methods to return the limits ?

Reply:
Since we are having a global tracer provider and span limits are associated with that only, so that should be better as it will simplify arguments passing. Will make the changes in next commit.